pkgs/by-name/pu/pushup/package.nix

```diff
@@ -0,0 +1,34 @@
+{ fetchFromGitHub, buildGoModule, lib, pkgs }:
+buildGoModule rec {
+  pname = "pushup";
+  version = "0.2";
+
+  src = fetchFromGitHub {
+    owner = "adhocteam";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-9ENXeVON2/Bt8oXnyVw+Vl0bPVPP7iFSyhxwc091ZIs=";
+  };
+
+  vendorHash = null;
+  subPackages = ".";
+  CGO_ENABLED = 0;
+  ldflags = [ "-s" "-w" ];
+  nativeBuildInputs = with pkgs; [ makeWrapper ];
+  allowGoReference = true;
+  postInstall = ''
+    wrapProgram $out/bin/${pname} --prefix PATH : ${
+      lib.makeBinPath (with pkgs; [ go ])
+    }
+  '';
+
+  meta = with lib; {
+    description = "Pushup web framework for Go";
+    homepage = "https://pushup.adhoc.dev/";
+    license = licenses.mit;
+    changelog =
+      "https://github.com/adhocteam/${pname}/blob/${src.rev}/CHANGELOG.md";
```

Comment:
this should be fine to have in one line

Reply:
@Janik-Haag Thank you for your review of my PR. I believe I have addressed your requests in the latest commits,